src/fragments/lib-v1/storage/js/copy.mdx

```diff
@@ -34,19 +31,19 @@ You can copy a file from the specified key to another key within the same [File
 ```javascript
 import { copy, list } from 'aws-amplify/storage';
 
-console.log(await list({ path : 'copied/' })); // []
+console.log(await list({ prefix : 'copied/' })); // []
```

Comment:
Hmmm, I think found this in previous PR but was not addressed🤔. It seems to lost during closing in-progress PR and creating new one. Let's try stick to 1 PR next time ☺️. cc @kvramyasri7

Reply:
yeah Sure.  There are some merge conflicts can surly do that next time